tasks/vg_map_hts.wdl

```diff
@@ -148,3 +149,274 @@ task extractReference {
         docker: "quay.io/vgteam/vg:ci-684-bc9aa5dfc4b0d14519ea47333075906a4ec74656"
     }
 }
+
+task createDistanceIndex {
+    input {
+        File in_gbz_file
+        Int in_extract_mem = 60 + 20
```

Comment:
Is there a good reason to express `80` like this? Was this supposed to involve a variable?